packages/react-instantsearch/src/connectors/connectMenu.js

```diff
@@ -28,7 +28,7 @@ function getValue(name, props, searchState) {
   return name === currentRefinement ? '' : name;
 }
 
-const sortBy = ['count:desc', 'name:asc'];
+const sortBy = ['isRefined', 'count:desc', 'name:asc'];
```

Comment:
We need to add with SFFV in order to keep track of the newly selected item. 
But for a regular menu it doesn't feel right to be the default behavior. 
WDYT?